internal/pkg/cli/pipeline_show.go

```diff
@@ -184,7 +184,9 @@ func (o *showPipelineOpts) askPipelineName() error {
 }
 
 func (o *showPipelineOpts) retrieveAllPipelines() ([]string, error) {
-	pipelines, err := o.pipelineSvc.ListPipelinesForProject(o.ProjectName())
+	pipelines, err := o.pipelineSvc.ListPipelineNamesByTags(map[string]string{
+		"ecs-project": o.ProjectName(),
```

Comment:
Will we be changing the ecs-project tag in a future pr?

Reply:
yes! we will. The plan is to change those tags once we get to the `stack` pkg